torch_glow/tests/nodes/abs_test.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import torch
+from tests import utils
+import unittest
```

Comment:
There're some internal linter complaints prevent landing. Would you mind apply my suggestion changes and I'll import again afterwards. Sorry for the inconvenience.

```suggestion

import unittest

import torch
from tests import utils
```

Reply:
No worries!